src/lib/statsig/statsig.tsx

```diff
@@ -26,19 +26,31 @@ type StatsigUser = {
     bundleDate: number
     refSrc: string
     refUrl: string
+    referrer: string
+    referrerHostname: string
     appLanguage: string
     contentLanguages: string[]
   }
 }
 
 let refSrc = ''
 let refUrl = ''
+let referrer = ''
+let referrerHostname = ''
 if (isWeb && typeof window !== 'undefined') {
   const params = new URLSearchParams(window.location.search)
   refSrc = params.get('ref_src') ?? ''
   refUrl = decodeURIComponent(params.get('ref_url') ?? '')
 }
 
+if (isWeb && document !== 'undefined' && document != null) {
```

Comment:
You want `typeof document !== 'undefined`, not `document !== 'undefined'`

Reply:
sigh lol, nice